mlir/tools/quantum-opt/quantum-opt.cpp

```diff
@@ -39,6 +40,7 @@ int main(int argc, char **argv)
 
     mlir::DialectRegistry registry;
     mlir::registerAllDialects(registry);
+    mlir::func::registerAllExtensions(registry);
```

Comment:
For my knowledge, is this a strong requirement from using the newer version of LLVM/MLIR?

Reply:
Yes, see [here](https://github.com/llvm/llvm-project/commit/a5ef51d786f241cdd9d1bec62fbc2bf89a766b7d).